<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePasswordDialog</class>
 <widget class="QDialog" name="ChangePasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Zmiana hasła</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/images/icon.ico</normaloff>:/images/images/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
background-color: rgb(255, 255, 255);
}


QMessageBox QPushButton:hover {
border-radius: 5px;
background: rgb(255,140,0);
}

QMessageBox QPushButton{
color: white;
border-radius: 5px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
   stop: 0 rgba(255,140,0), stop: 0.7 rgb(255,105,0));
min-height: 30px;
min-width: 70px;
}

QMessageBox QPushButton:pressed {
color: white;
border-radius: 5px;
background: rgb(255,105,0);
}
</string>
  </property>
  <widget class="QLineEdit" name="leOldPassword">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>251</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: gray;</string>
   </property>
   <property name="text">
    <string>Wprowadź nowe hasło:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNewPassword">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>251</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: gray;</string>
   </property>
   <property name="text">
    <string>Wprowadź aktualne hasło:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: gray;</string>
   </property>
   <property name="text">
    <string>Potwierdź nowe hasło:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNewPasswordAgain">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>251</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConfirm">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>105</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
border-radius: 5px;
background: rgb(0,190,0);
}

QPushButton{
color: white;
border-radius: 5px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
   stop: 0 rgba(0,190,0), stop: 0.7 rgb(0,150,0));
}

QPushButton:pressed {
color: white;
border-radius: 5px;
background: rgb(0,150,0);
}
</string>
   </property>
   <property name="text">
    <string>Zatwierdź</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>105</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover {
border-radius: 5px;
background: rgb(255,102,102);
}

QPushButton{
color: white;
border-radius: 5px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
   stop: 0 rgba(255,102,102), stop: 0.7 rgb(255,50,50));
}

QPushButton:pressed {
color: white;
border-radius: 5px;
background: rgb(255,50,50);
}
</string>
   </property>
   <property name="text">
    <string>Anuluj</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
